build/target-repository/docs/rector_rules_overview.md

```diff
@@ -6906,6 +6906,40 @@ return static function (RectorConfig $rectorConfig): void {
 
 <br>
 
+### RemoveFuncCallRector
+
+Remove function
+
+:wrench: **configure it!**
+
+- class: [`Rector\Removing\Rector\FuncCall\RemoveFuncCallRector`](../rules/Removing/Rector/FuncCall/RemoveFuncCallRector.php)
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Removing\Rector\FuncCall\RemoveFuncCallRector;
+use Rector\Removing\ValueObject\RemoveFuncCall;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(RemoveFuncCallRector::class, [
+        new RemoveFuncCall('var_dump'),
```

Comment:
please run `composer docs` to regenerate this as use array of string

Reply:
Sorry for the back and forth, and thank you so much for your feedback :D